pkg/api/requests.go

```diff
@@ -15,6 +18,16 @@ type Resources struct {
 	NodeList             *k8sapicore.NodeList
 	StorageClassList     *k8sapistorage.StorageClassList
 	NamespaceList        []NamespaceResources
+	RBACResources        RBACResources
```

Comment:
Just wondering why not listing Users, Groups, Roles, and SecurityContextConstraint as such instead of wrapping them in RBAC?

Reply:
I my opinion things related to RBAC should go under separate attribute for keeping obvious to which part of report user, group, etc belong. It will be easier later to parse json in UI.

Reply:
I understand the choice just that I'm not sure about it.